getting_started/first_3d_game/01.game_setup.rst

```diff
@@ -30,7 +30,7 @@ Click *Import & Edit* to open the project in the editor.
 The start project contains an icon and two folders: ``art/`` and ``fonts/``.
 There, you will find the art assets and music we'll use in the game.
 
-.. image:: img/01.game_setup/04.start_assets.png
```

Comment:
This file was updated, see below, to reference the newer filenames base don the 4.x demo.